app/scripts/modules/core/src/plugins/deck.plugin.ts

```diff
@@ -11,8 +11,10 @@ export interface IDeckPlugin {
   stages?: IStageTypeConfig[];
   /** Custom Preconfigured Job Stage UI (configuration and execution details) */
   preconfiguredJobStages?: IStageTypeConfig[];
-  /** Custom managed resource kinds */
+  /** DEPRECATED - Custom managed resource kinds */
   resourceKinds?: IResourceKindConfig[];
+  /** Managed Delivery hooks */
+  managedDelivery?: IManagedDeliveryPlugin;
```

Comment:
could you make a case for packaging more than one MD plugin into a single `IDeckPlugin`?

Reply:
You can have multiple `resources` and `constraints` within a single `managedDelivery`. I can't think of a reason to have more the one `managedDelivery` if that's what you mean